.github/workflows/publish.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         uses: "gradio-app/gradio/.github/actions/install-all-deps@main"
         with:
           always_install_pnpm: true
+          python_version: "3.10"
```

Comment:
I have to manually specify this b/c our CI uses the version of the action that is on `main`, and the one currently on `main` is using 3.8